tools/CMakeLists.txt

```diff
@@ -1,7 +1,12 @@
-add_executable(
-	HogMaker
-	HogMaker/HogFormat.cpp
-	HogMaker/HogMaker.cpp
-)
-add_dependencies(HogMaker get_git_hash)
-target_include_directories(HogMaker PRIVATE ${PROJECT_BINARY_DIR}/lib)
+if (ANDROID)
+	find_package(HogMaker REQUIRED)
+else()
+	add_executable(
+		HogMaker
+		HogMaker/HogFormat.cpp
+		HogMaker/HogMaker.cpp
+	)
+	export(TARGETS HogMaker FILE "${CMAKE_BINARY_DIR}/HogMakerConfig.cmake")
```

Comment:
HogMakerConfig.cmake was not included in this PR. And in this else(), we're not targeting Android, which means that HogMaker will not be looked up using `find_package`. So why exporting the target?